From: Jeroen Roovers <jer@gentoo.org>
Subject: add parisc support to path_id

As reported at http://bugs.gentoo.org/show_bug.cgi?id=113379 path_id doesn't
work too well for parisc boxes.  This patch from Jeroen Roovers
<jer@gentoo.org> fixes that.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff --git a/extras/path_id b/extras/path_id
index 4ac7187..dfb997b 100755
--- a/extras/path_id
+++ b/extras/path_id
@@ -235,6 +235,11 @@ handle_block_scsi () {
 		adapter="`echo $controller_dev |  sed -e 's@/[^/]\{1,\}$@@;s@^.*/@@;s@^.*:@@'`"
 		bus="gc"
 		;;
+		# PARISC devices
+		*parisc*)
+		adapter="${controler_dev##*/}"
+		bus=parisc;
+		;;
 		*)
 		adapter="${controller_dev##*/}"
 		bus="pci"
